code/game/machinery/syndicatebomb.dm

```diff
@@ -449,6 +455,27 @@
 	range_light = 4
 	range_flame = 2
 
+/obj/item/bombcore/emp
+	name = "EMP bomb core"
+	var/light_emp = 36
+	var/heavy_emp = 18
+	var/pulse_number = 1 //Since one EMP wont destroy anything other then consoles and IPCS, here is an option to have multiple pulses when dentonating. DO NOT USE THIS WITH REALLY LARGE AREAS
+	var/adminloged = FALSE //If it exploded once, don't do it again.
+
+/obj/item/bombcore/emp/detonate()
+	if(adminlog && !adminloged)
+		message_admins(adminlog)
+		log_game(adminlog)
+		adminloged = TRUE
+	empulse(src, heavy_emp, light_emp, 1)
+	if(pulse_number <= 1)
+		if(loc && istype(loc, /obj/machinery/syndicatebomb))
+			qdel(loc)
+		qdel(src)
+	else
+		pulse_number -= 1
+		addtimer(CALLBACK(src, .proc/detonate), 20) // every 2 seconds go off again till pulses run out
```

Comment:
Add some indication to the bombcore that it is burnt out, perhaps?

Reply:
Sure, one moment